src/main/kotlin/it/polpetta/utils/JenkinsSession.kt

```diff
@@ -26,8 +26,8 @@ import it.polpetta.config.Resources
 import java.io.FileNotFoundException
 import java.nio.file.Path
 
-object Jenkins {
-    val session: JenkinsClient?
+class JenkinsSession {
```

Comment:
Can't we use objects anymore with Guice?

Reply:
No we actually can, i didn't remember to change it back to object.
On a side note, objects cannot have a constructor, therefore no dependencies can be injected, which might make them less usefult